ui/components/app/transaction-activity-log/transaction-activity-log-icon/transaction-activity-log-icon.stories.js

```diff
@@ -14,6 +14,9 @@ export default {
       options: Object.keys(imageHash),
     },
   },
+  args: {
+    eventKey: 'TRANSACTION_CREATED_EVENT',
+  },
 };
 
 export const DefaultStory = (args) => <TransactionActivityLogIcon {...args} />;
```

Comment:
Setting a default so it's not a blank screen

<img width="895" alt="Screen Shot 2022-05-17 at 2 35 43 PM" src="https://user-images.githubusercontent.com/8112138/168914129-f850da34-c52f-46a7-9247-cde85335f1bc.png">